ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -125,6 +125,8 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 			 */
 			offset: PropTypes.oneOf(['none', 'overlap', 'small']),
 
+			onAdjustDirection: PropTypes.func,
```

Comment:
When you add a new prop, jsdoc needs to be added.

Reply:
I added jsdoc.
But I don't sure some params are correct. They need to be checked.

Reply:
Could you please make this prop private?

Reply:
I accepted your suggestion.

Reply:
@vJIYEv @juwonjeong Even if the prop name is private, please discuss with members next time if the name looks fine. BTW, onAdjustDirection looks good to me :)
CC: @0x64